apps/server/src/api-integration/integration.utils.ts

```diff
@@ -41,19 +43,71 @@ const propertyConversion = {
   timeEnd: (value: unknown) => clampDuration(coerceNumber(value)),
 };
 
-export function parseProperty(property: string, value: unknown) {
+export function parseEventProperty(property: string, value: unknown) {
```

Comment:
question: I didnt understand the need to have different maps per entry type. The same property should have the same type in all entries, is that not right?

Reply:
there are not timers in milestone and group
and the group has the targetDuration